src/tm-threads.c

```diff
@@ -248,7 +253,14 @@ static void *TmThreadsSlotPktAcqLoop(void *td)
                                  " tmqh_out=%p",
                    s, s ? s->PktAcqLoop : NULL, tv->tmqh_in, tv->tmqh_out);
         TmThreadsSetFlag(tv, THV_CLOSED | THV_RUNNING_DONE);
-        pthread_exit((void *) -1);
+#ifdef HAVE_DPDK
+        if (tv->lcore_id == LCORE_ID_ANY) {
```

Comment:
What is happening here ? I think this deserves a comment

Reply:
In #6269 I have tried to be more descriptive at all places where LCORE_ID_ANY is used. I also did a bit of refactoring. Thanks for pointing this out.